### PR TITLE
Merging ufcg-lsd/vertical-cpu into ufcg-lsd/master

### PR DESCRIPTION
**Pull request by rafaelvfalc**
- Manager version that support both kubejobs and vertical plugins
- Update of the documentation

### CODE REVIEW

**Comment by javanlacerda** [comment]
I think that is better remove the sensitive dates from broker.cfg.

File: broker/plugins/kubejobs/plugin.py [hunk 1]
```diff
@@ -104,18 +142,23 @@ def start_application(self, data):
...
+
+            time.sleep(float(30))
+
+            if self.enable_visualizer:
+                visualizer.stop_visualization(api.visualizer_url,
+                                                self.app_id, data['visualizer_info'])
             monitor.stop_monitor(api.monitor_url, self.app_id)
             controller.stop_controller(api.controller_url, self.app_id)
-            print "stoped services"
+            print "Stoped services"
```
Comment (clenimar) [inline broker/plugins/kubejobs/plugin.py]: typo: Stopped.

File: docs/plugins/kubejobs.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+# KubeJobs Plugin
+
+## How does it works?
```
Comment (clenimar) [inline docs/plugins/kubejobs.md]: typo: work.

**Review by clenimar — APPROVED** [review]
just a few non-blocker typos.

**Pull request merged**